common/utils.go

```diff
@@ -564,16 +576,6 @@ func NormalizeForEnvironmentVariableName(envName string) string {
 	return newName
 }
 
-// IsStringPresent checks if a value is present in a slice
-func IsStringPresent(list []string, value string) bool {
```

Comment:
IsStringPresent does a case insensitive comparison, that's why we are not able to use generic for strings and ints.

Reply:
It might be ok to sacrifice the flexibility there to gain clarity in code and usage.